src/main/java/com/netgrif/application/engine/importer/service/Importer.java

```diff
@@ -485,7 +485,7 @@ protected void createTransition(com.netgrif.application.engine.importer.model.Tr
 
         Transition transition = new Transition();
         transition.setImportId(importTransition.getId());
-        transition.setTitle(toI18NString(importTransition.getLabel()));
+        transition.setTitle(toI18NString(importTransition.getLabel()) != null ? toI18NString(importTransition.getLabel()) : new I18nString(""));
```

Comment:
It would be more readable if the null check was performed on the `getLabel()` call and not on the `toI18NString()` call

Reply:
Done